react/features/filmstrip/actions.web.js

```diff
@@ -435,3 +436,16 @@ export function setStageParticipants(queue) {
         queue
     };
 }
+
+/**
+ * Sets the max number of participants to be displayed on stage.
+ *
+ * @param {number} maxParticipants - Max number of participants.
+ * @returns {Object}
+ */
+export function setMaxStageParticipants(maxParticipants) {
+    return {
+        type: SET_MAX_STAGE_PARTICIPANTS,
+        maxParticipants
+    };
```

Comment:
Shouldn't this line consider the current maxParticipants?: 

https://github.com/jitsi/jitsi-meet/blob/42703fed474c72606f21e92214a24a0b1b5aaed7/react/features/filmstrip/actions.web.js#L268

Reply:
Not really. There it's used as an absolute max kind of thing, so it's ok to leave it like this